File: SFA-API/Domain/ShopCategory.cs
using System.Collections.Generic;

namespace Domain
{
    public class ShopCategory
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string ShopCategoryCode { get; set; }
        public decimal MaximumDebtAmount { get; set; }
        public virtual List<Shop> Shops { get; set; }
    }
}

File: SFA-Client/src/app/_models/shopCategory.ts
export class ShopCategory {
  id: number;
  name: string;
  description: string;
  shopCategoryCode: string;
  maximumDebtAmount: number;

  constructor(shopCategory) {
    this.id = shopCategory.id;
    this.name = shopCategory.name;
    this.description = shopCategory.description;
    this.shopCategoryCode = shopCategory.shopCategoryCode;
    this.maximumDebtAmount = shopCategory.maximumDebtAmount;
  }
}
